applications/ExaquteSandboxApplication/tests/test_ExaquteSandboxApplication.py

```diff
@@ -28,6 +30,10 @@ def AssembleTestSuites():
     # - testSmallExample
     smallSuite = suites['small']
     smallSuite.addTest(KratosExaquteSandboxGeneralTests('testSmallExample'))
+    if(hasattr(MeshingApplication,"MmgProcess2D")):
+        smallSuite.addTest(TimeAveragingProcessTests('testDivergenceFreeRefinementProcess'))
+    else:
+        print("MMG process is not compiled and the corresponding tests will not be executed")
```

Comment:
I think that it's better to do this check in the specific test, using the `self.skip_test` method

For instance:
https://github.com/KratosMultiphysics/Kratos/blob/24a655a28a0c8aca15c6d85f7da1e2986affa5d8/applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py#L43-L45

With your current code, the test suite wouldn't recognize it as an skipped test I think.

Reply:
Done